.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

I would change this to Tuesday as this aligns better with the start of a new issue supervisor.
```suggestion
      interval: "tuesday"
```